project/locations/animal_rescue/animal_rescue.tscn

```diff
@@ -1,51 +1,239 @@
-[gd_scene load_steps=9 format=3 uid="uid://dgvmjpupptnko"]
+[gd_scene load_steps=13 format=4 uid="uid://dgvmjpupptnko"]
 
-[ext_resource type="Texture2D" uid="uid://dytiuen0u3y8q" path="res://locations/animal_rescue/art/animal shelter.png" id="1_jc286"]
+[ext_resource type="Texture2D" uid="uid://c76afxxde4c3o" path="res://locations/animal_rescue/art/animal shelter_atlas.png" id="1_h46rp"]
 [ext_resource type="PackedScene" uid="uid://c3iur4l67uyh2" path="res://characters/player_character/player_character.tscn" id="1_mdjag"]
-[ext_resource type="PackedScene" uid="uid://cjm1wa8lbybab" path="res://characters/npcs/german_shepard/german_shephard.tscn" id="5_i1vl6"]
-[ext_resource type="PackedScene" uid="uid://bb4geg1u4xjjp" path="res://characters/npcs/long_dog/long_dog.tscn" id="6_qkjpx"]
-[ext_resource type="PackedScene" uid="uid://dg3c4ngx1j6bk" path="res://characters/npcs/persian_cat/persian_cat.tscn" id="7_5rebn"]
-[ext_resource type="Texture2D" uid="uid://cybq4tp15n2fj" path="res://locations/animal_rescue/art/shelter_clerk_world.png" id="7_hny44"]
-[ext_resource type="Texture2D" uid="uid://c00mw7n1sqlwu" path="res://locations/animal_rescue/art/desk.png" id="8_kpnvr"]
-[ext_resource type="PackedScene" uid="uid://ciq52ea2psb3d" path="res://characters/npcs/pug_dog/pug_dog.tscn" id="8_xb5q7"]
+[ext_resource type="Texture2D" uid="uid://dqepo38i37jrk" path="res://locations/animal_rescue/art/animal shelter.png" id="2_74spk"]
+[ext_resource type="TileSet" uid="uid://cej2r4jp8fdt6" path="res://locations/animal_rescue/art/animal shelter_sprites.tres" id="2_gfjcx"]
+[ext_resource type="Texture2D" uid="uid://pkbq6gvrrbew" path="res://characters/npcs/shelter_clerk/art/shelter_clerk.png" id="3_01e37"]
+[ext_resource type="SpriteFrames" uid="uid://txq4sdir51xh" path="res://characters/npcs/german_shepard/german shepard more diarrhea.res" id="3_ryi3h"]
+[ext_resource type="Texture2D" uid="uid://buhydr0ohph0f" path="res://characters/npcs/long_dog/longDog.png" id="4_8ix7j"]
+[ext_resource type="Texture2D" uid="uid://b2dto3uagjkyx" path="res://characters/npcs/persian_cat/Persian Cat.png" id="5_rcah5"]
+[ext_resource type="Texture2D" uid="uid://dnu263ojg426r" path="res://characters/npcs/pug_dog/pug dog.png" id="6_t4ses"]
```

Comment:
@shiser 

It looks like this change would revert the reference to the persian cat scene at the shelter to the Texture2D that ist was previously.

Do you feel you are able to make a change to this so that it incorporates your work but doesn't revert those animal character scenes?

If you need any help or suggestions, let me know. It's no problem!